src/bolos/cx_scc.c

```diff
@@ -2,6 +2,8 @@
 
 #include "bolos/exception.h"
 #include "cx.h"
+#include "cx_hmac.h"
+
```

Comment:
This blank line makes the CI [fail](https://github.com/LedgerHQ/speculos/pull/141/checks?check_run_id=2079398603), please delete it.

Reply:
done